packages/bff/src/providers/clientProvider.ts

```diff
@@ -1,30 +1,54 @@
-import { createApiClient as createApiClientTenantProcess } from "../model/generated/tenant-process/api.js";
-import { createApiClient as createApiClientAgreementProcess } from "../model/generated/agreement-process/api.js";
-import { createApiClient as createApiClientCatalogProcess } from "../model/generated/catalog-process/api.js";
-import { createApiClient as createApiClientAttributeProcess } from "../model/generated/attribute-process/api.js";
-import { createApiClient as createApiClientPurposeProcess } from "../model/generated/purpose-process/api.js";
+import {
+  tenantApi,
+  attributeRegistryApi,
+  catalogApi,
+  agreementApi,
+  purposeApi,
+} from "pagopa-interop-api-clients";
 import { config } from "../utilities/config.js";
 
+type TenantProcessClient = {
+  tenants: ReturnType<typeof tenantApi.createTenantApiClient>;
+};
+
+type AttributeProcessClient = ReturnType<
+  typeof attributeRegistryApi.createAttributeApiClient
+>;
+
+type CatalogProcessClient = ReturnType<
+  typeof catalogApi.createProcessApiClient
+>;
+
+type AgreementProcessClient = ReturnType<
+  typeof agreementApi.createAgreementApiClient
+>;
+
+type PurposeProcessClient = ReturnType<
+  typeof purposeApi.createPurposeApiClient
+>;
+
 export type PagoPAInteropBeClients = {
-  tenantProcessClient: ReturnType<typeof createApiClientTenantProcess>;
-  attributeProcessClient: ReturnType<typeof createApiClientAttributeProcess>;
-  catalogProcessClient: ReturnType<typeof createApiClientCatalogProcess>;
-  agreementProcessClient: ReturnType<typeof createApiClientAgreementProcess>;
-  purposeProcessClient: ReturnType<typeof createApiClientPurposeProcess>;
+  tenantProcessClient: TenantProcessClient;
+  attributeProcessClient: AttributeProcessClient;
+  catalogProcessClient: CatalogProcessClient;
+  agreementProcessClient: AgreementProcessClient;
+  purposeProcessClient: PurposeProcessClient;
 };
 
 export function getInteropBeClients(): PagoPAInteropBeClients {
   return {
-    tenantProcessClient: createApiClientTenantProcess(config.tenantProcessUrl),
-    agreementProcessClient: createApiClientAgreementProcess(
+    tenantProcessClient: {
+      tenants: tenantApi.createTenantApiClient(config.tenantProcessUrl),
+    },
```

Comment:
In this case (and possibly in others) we have more than one client (actually one for each tag). In the future, we should expose every possible client divided by tag